WTWSMS/localisation/WtWSMS_titles.csv

```diff
@@ -1054,6 +1055,7 @@ c_trading_company_adj;Tradesman;;;;;;;;;;;x
 TRADING_COMPANY_TITLE;Admiral;;;;;;;;;;;x
 d_socotra;Socotra;;;;;;;;;;;x
 d_socotra_adj;Socotri;Socotri;Socotri;;Socotri;;;;;;;;;x
+b_xanten;Xanten;;;;;;;;;;;x
```

Comment:
I don't understand this. The mod doesn't add any barony of Xanten.

Reply:
@tannerflick4 Changed b_Xanten to b_xanten IIRC

Reply:
Uh, okay. So you are doing this preemptively? Oh, whatever, it doesn't matter.

Reply:
No, this was reported by the Validator, and IIRC it's already changed in master

Reply:
Okay, I understand